synapseclient/__main__.py

```diff
@@ -225,7 +226,7 @@ def associate(args, syn):
     for fp in files:
         try:
             ent = syn.get(fp, limitSearch=args.limitSearch)
-        except SynapseFileNotFoundError:
+        except synapseclient.core.models.SynapseFileNotFoundError:
```

Comment:
let's pulll the Erros into their own package (`synapseclient.errors.SynapseError`)